### DISCUSSION TITLE
My github account was stolen

### DISCUSSION THREAD

**Discussion by ernklyc** [community/community#136386] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

On August 19, my account with the email address  was stolen by changing the dive password. Can you help me get my account back?

**Comment by pierrelouisguichard** [comment, marked as answer]
Sorry to hear that, your best course of action is to contact github support they will be able to help you recover your account.
[Github Support](https://docs.github.com/en/support/contacting-github-support)
